### ISSUE TITLE
In membership sites: Possibility to turn off download for individual media files

### ISSUE THREAD

**Issue by ingjerd123** [Simplero/Roadmap#248]
I allow my students to download lesson videos, etc. But they have no need for downloading a banner image, welcome videos, etc. In spaces you can turn off downloads for individual media files, I wish that was possible in membership sites too so that download text don't disturb the visual

<img width="1203" alt="skjermbilde 2018-01-20 kl 22 16 56" src="https://user-images.githubusercontent.com/26975740/35190438-dd7ce03a-fe2f-11e7-83cc-5220d4027d1e.png">

**Comment by calvincorreli** [comment]
Maybe we just never offer download option for images?

**Comment by ingjerd123** [comment]
Yep, If someone wants to download the pictures, they can do that anyway

There are not all videos there are any point in downloading, but I guess it doesn't matter if the option is there

**Comment by tueolesen** [comment]
Depending on people's browser, videos may also be download-able even when downloads are disabled in simplero.

**Comment by nicolabird** [comment]
I would love to be able to de-select any media item individually to prevent it being downloaded.
